### ISSUE TITLE
Button customization did not work properly

### ISSUE THREAD

**Issue by ThomAce** [christophhart/HISE#554]
Hi,

In the latest version, the button customization did not work at all. Does not matter which way I try to make it (followed the online video exactly the same way), in the latest version it does nothing on Windows. It is also an interesting behavior when resizing the button, the radio-button graphic starts to move to the center of the button (in case of certain square-like shape) and hovering on top of the button text.

**Comment by davidhealey** [comment]
> In the latest version, the button customization did not work at all.

What do you mean by button customization?

> followed the online video exactly the same way)

Which video?

**Comment by ThomAce** [comment]
This is what I'm referring to: https://www.youtube.com/watch?v=badkm6QVNIA

**Comment by davidhealey** [comment]
Post a snippet that isn't working and I'll take a look at it.

**Comment by ThomAce** [comment]
Figured out that there were some bugs in installation. Reinstalled HISE and this problem just gone. I followed this video where the text and toggle types did not worked while the icon types worked well. https://youtu.be/badkm6QVNIA?si=XCjVUUXgYm34S_RR&t=980

The more better representation I found is on this page. https://docs.hise.dev/glossary/custom_lookandfeel.html#getting-started
Actually, this helped a lot.

Thanks!
